launch/sim_iarc7_safety.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
```

Comment:
I agree with all of that, the only thing I'm not sure about is whether we should pass an argument that holds the path to the yaml file or just one that indicates which mode to run in